python/ray/train/huggingface/huggingface_trainer.py

```diff
@@ -120,7 +120,11 @@ class HuggingFaceTrainer(TorchTrainer):
     This Trainer runs the ``transformers.Trainer.train()`` method on multiple
     Ray Actors. The training is carried out in a distributed fashion through PyTorch
     DDP. These actors already have the necessary torch process group already
-    configured for distributed PyTorch training.
+    configured for distributed PyTorch training. If you have PyTorch >= 1.12.0
+    installed, you can also run FSDP training by specifying the ``fsdp`` argument
+    in ``TrainingArguments``. For more information on configuring FSDP,
+    refer to `Hugging Face documentation <https://huggingface.co/docs/transformers/\
+main/en/main_classes/trainer#transformers.TrainingArguments.fsdp>`__.
```

Comment:
Also seems like the .fsdp doesn't work- it just redirects me to the main Trainer doc

```suggestion
main/en/main_classes/trainer#transformers.TrainingArguments>`__.
```